frontend/src/components/UI/USWDS/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,41 @@
+import React, {Fragment, useLayoutEffect, useRef} from "react"
+import PropTypes from "prop-types";
+import Accordion from "../../Accordion";
+import USWDS from "@uswds/uswds/js";
+
+const { tooltip } = USWDS;
+
+/**
+ * A tooltip is a short descriptive message that appears when a user hovers or focuses on an element.
+ *
+ * @param {string} label - the content of the tooltip
```

Comment:
think we need to add `props` here like this example"

```js
* @param {object} props - The component props.
* @param {object} props.agreement - The agreement object to display details for.
 ```

Reply:
done